routers/repo/issue.go

```diff
@@ -206,17 +206,29 @@ func Issues(ctx *context.Context) {
 		}
 	}
 
+	var issuesStates = make([]*models.CommitStatus, 0, len(issues))
 	// Get posters.
-	for i := range issues {
+	for i, issue := range issues {
 		// Check read status
 		if !ctx.IsSigned {
 			issues[i].IsRead = true
 		} else if err = issues[i].GetIsRead(ctx.User.ID); err != nil {
 			ctx.Handle(500, "GetIsRead", err)
 			return
 		}
+
+		if issue.IsPull {
+			issue.LoadAttributes()
```

Comment:
Let's not ignore the error returned here

Reply:
done.